spring-boot-autoconfigure/src/main/java/org/axonframework/springboot/connection/AxonServerTestContainerConnectionDetailsFactory.java

```diff
@@ -0,0 +1,35 @@
+package org.axonframework.springboot.connection;
```

Comment:
I am not entirely suer about the package name, but I don't have another suggestion either...

Reply:
I will mimic Spring's naming: `org.axonframework.springboot.service.connection`

Reply:
That sounds good to me, thanks :-)